share/spack/docker/README.md

```diff
@@ -0,0 +1,20 @@
+Using this docker container for development.
+------------
+
+    $ cd share/spack/docker
```

Comment:
if you like you can add syntax highlighting for bash [like this](https://help.github.com/articles/creating-and-highlighting-code-blocks/#syntax-highlighting) in markdown